### ISSUE TITLE
Feature Request: Live Notifications for Scheduled Videos

### ISSUE THREAD

**Issue by neo-neo1** [anxdpanic/plugin.video.youtube#796]
I don't know if it's possible but having the ability to enable Kodi notifications for future scheduled live streams would be nice to have.

I'm thinking about these 2 options:

1). Enable notification per single individual future scheduled live video.

2). Enable all live notifications per specific channel. 

Thanks in advance

**Comment by MoojMidge** [comment]
> would be nice to have.

Agree, but not something I use personally, so not high on the priority list. More fundamental things needs to get done first.

> 1). Enable notification per single individual future scheduled live video.

In concept, should be fairly straightforward to do.



> 2). Enable all live notifications per specific channel.

Not so straightforward, and probably won't try to do this.

**Comment by MoojMidge** [comment]
Looking at this now. Live notifications for all new videos in a subscribed/bookmarked channel should be possible.

Please provide further details of how this should work e.g.

- What should the notification show?
- What should the notification do?
  - Should it only be a notification (i.e. a reminder) or should it be a dialog that prompts to play, dismiss, or remind later?
  - Should the notification open at the scheduled playback time, or earlier?
  - Is there a comparable feature in other plugins or video players?

**Comment by MoojMidge** [comment]
Is there any interest in this?

It's a hassle to keep rebasing this. Will need to drop this if it is not going to progress anywhere further.

**Comment by neo-neo1** [comment]
Feel free to delete if it's too time consuming.

While it's a nice addition to have, that time would be better spent to fix existing problems such as queue/playlist bugs and my subscription favorite caching.

**Comment by MoojMidge** [comment]
This is wating on you... you requested the feature but haven't identified how you actually want it to work. https://github.com/anxdpanic/plugin.video.youtube/issues/796#issuecomment-2211630895

Not aware of any other existing problems. If you have issues then you need to provide details if you want them fixed.
